src/resources/views/auth/signin.blade.php

```diff
@@ -4,27 +4,46 @@
 
 @section('content')
     <div class="form-container">
+        <div class="user-details">
+            @auth
+            <br>
+            *Log out for testing purposes only*
+            {{ auth()->user()->email}}
+            <br>
+            <form action="/logout" method="GET">
+            </div>
+            <button class="button">Log Out</button>
+            </form>
+        @else
         <div class="title">
             <h3>Sign In</h3>
         </div>
-        <form action="#">
+        <form action="/login" method='POST'>
+            @csrf
             <div class="user-details">
                 <div class="input-box">
-                    <input type="text" autocomplete="off" required="required">
+                    <input type="text" name='username' id="username" value="{{ old('username') }}" autocomplete="off" required="required">
                     <span>Username</span>
+                    @error('username')
+                    <div style="color: red;">{{ $message }}</div>
+                    @enderror
                 </div>
 
                 <div class="input-box">
-                    <input type="password" required="required">
+                    <input type="password" type="password" name='password' id="password" required="required">
```

Comment:
refrain from using HTML5 validation. You can handle the validation in backend via Requests or frontend validation using JS